site/src/xServices/audit/auditXService.ts

```diff
@@ -0,0 +1,44 @@
+import { getAuditLogs } from "api/api"
+import { assign, createMachine } from "xstate"
+
+type AuditLogs = Awaited<ReturnType<typeof getAuditLogs>>
+
+export const auditMachine = createMachine(
+  {
+    id: "auditMachine",
+    schema: {
+      context: {} as { auditLogs?: AuditLogs },
+      services: {} as {
+        loadAuditLogs: {
+          data: AuditLogs
+        }
+      },
+    },
+    tsTypes: {} as import("./auditXService.typegen").Typegen0,
+    initial: "loadingLogs",
+    states: {
+      loadingLogs: {
+        invoke: {
+          src: "loadAuditLogs",
+          onDone: {
+            target: "success",
+            actions: ["assignAuditLogs"],
```

Comment:
What happens in case of error?

Reply:
I forgot to handle the error 🤦 going to fix

Reply:
Added.